Dockerfile

```diff
@@ -1,10 +1,17 @@
-FROM quay.io/ukhomeofficedigital/nodejs-base:v6.9.1
+FROM quay.io/ukhomeofficedigital/nodejs-base:v6.11.0
 
 RUN mkdir /public
+RUN yum clean all && \
+  yum update -y -q && \
+  yum clean all && \
+  rpm --rebuilddb
```

Comment:
umm nope but we should probably add the onbuild there...

https://github.com/UKHomeOffice/docker-nodejs-base/blob/master/Dockerfile